astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1886,7 +1901,9 @@ def __init__(
         self.is_dataclass: bool = False
         """Whether this class is a dataclass."""
 
-        self.type_params: list[nodes.TypeVar, nodes.ParamSpec, nodes.TypeVarTuple] = []
+        self.type_params: list[
+            nodes.TypeVar | nodes.ParamSpec | nodes.TypeVarTuple
+        ] = []
```

Comment:
?

Reply:
<img width="705" alt="Scherm­afbeelding 2023-07-02 om 22 14 36" src="https://github.com/pylint-dev/astroid/assets/13665637/4770dce2-1021-49c7-a6f7-3f33e801bfd1">

Reply:
Oh right, duh. I just literally couldn't see the change in the diff, I thought it was just a line break change. Sorry!